packages/sanity/src/core/releases/tool/detail/useBundleDocuments.ts

```diff
@@ -0,0 +1,51 @@
+import {isValidationErrorMarker} from '@sanity/types'
+import {useMemo} from 'react'
+import {useObservable} from 'react-rx'
+import {combineLatest} from 'rxjs'
+import {filter, map} from 'rxjs/operators'
+import {mergeMapArray} from 'rxjs-mergemap-array'
+
+import {useSchema} from '../../../hooks'
+import {useDocumentPreviewStore} from '../../../store'
+import {useSource} from '../../../studio'
+import {validateDocumentWithReferences} from '../../../validation'
+
+export function useBundleDocuments(bundle: string) {
+  const groqFilter = `defined(_version) &&  _id in path("${bundle}.*")`
```

Comment:
What do you think about requesting to content lake another function that works in a similar way to `(sanity::versionOf("${publishedId}")` but that returns all the bundle documents?

Reply:
Ohhh it's using path because it should fetch here the documents that belong to a bundle.
example:
bundle: summer-drop
it needs to fetch: `summer-drop.foo` and `summer-drop.bar` 

`sanity:versionOf` get's only the versions of a document.
So if we use:
`sanity:versionOf(foo)` it will return `[{_id: foo}, {_id: summer-drop.foo}]`

Reply:
ah, right! reverted in [757d430](https://github.com/sanity-io/sanity/pull/7257/commits/757d4304c69ed296142e5b51815efb12ac9c8206)